packages/flutter/test/widget/bottom_sheet_test.dart

```diff
@@ -23,6 +23,8 @@ void main() {
     await tester.pump();
     expect(find.text('BottomSheet'), findsNothing);
 
+    showBottomSheetThenCalled = false;
+    // ignore: unawaited_futures
```

Comment:
wait, what's the future that isn't awaited? the future has a `.then()` and everything.